Admin/AbstractAdmin.php

```diff
@@ -48,7 +48,7 @@
 use Symfony\Component\Validator\ValidatorInterface as LegacyValidatorInterface;
 
 /**
- * @author  Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
```

Comment:
I wonder, why this isn't fixed by PHP-CS-Fixer 😕 

//cc @greg0ire @Soullivaneuh